src/vm/i386/gmsx86.cpp

```diff
@@ -683,6 +683,10 @@ void LazyMachState::unwindLazyState(LazyMachState* baseState,
                 ip += 2;
                 break;
 
+            case 0x34:                            // XOR AL, imm8
```

Comment:
Can you please put both changes in this file under FEATURE_PAL ifdef? I would prefer not to change the behavior for Windows here.